_posts/2016-03-21-an-event-apart-nashville.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
+title: "An Event Apart Nashville: post-conference thoughts"
+date: 2016-03-21
+author: Anne Tomasevich
+tags: conference design front-end-dev
+summary: Some recap and reflection after my first An Event Apart conference.
+hero_image: "assets/img/aea_nashville_hero.png"
+hero_alt: "An Event Apart: Nashville logo over Broadway Street"
+
+---
+
+Sitting in the airport after attending [An Event Apart](http://aneventapart.com/)
+Nashville, I found myself feeling overwhelmed at the sheer volume of
+information I'd just taken in and the positively enormous possibilities lying
+ahead of the people who build the web (and the equally huge responsibility we
+have to move forward with thoughtfulness and compassion).
+
+I wasn't quite sure what to expect going in, so I'd like to summarize my biggest
+takeaways and share some of the things I found exciting.
+
+### Major takeaways
+
+Knowing the kind of ground-breaking conversations that can happen at An Event
+Apart (for instance, the idea of responsive design was introduced by
+[Ethan Marcotte](http://ethanmarcotte.com/) at
+[An Event Apart Seattle 2010](http://aneventapart.com/news/post/five-years-ago-today-responsive-web-designs-debut))
+I was hoping for a glimpse into the future of the web, and I wasn't disappointed.
+Throughout the event, a few overarching themes came up.
+
+#### Unified design is the logical continuation of responsive design.
+
+Between the growing capabilities of devices, the sheer number of models, the
+differences in how we use them depending on age/location/race/preference/ability,
+and our ever-increasing use of small screens, there is no "mobile" and "desktop"
+anymore. In his talk, [Cameron Moll](http://cameronmoll.tumblr.com/) replaced
+the responsive design paradigm with **unified design** - a cohesive, consistent
+experience across all devices regardless of where the experience begins,
+continues, and ends. If we commit to this, we empower users to not only have a
+seamless experience using our products, we also allow them to use whichever
+device makes the most sense for them. As Cameron put it:
+
+> The best interface is the one within reach.
+
+Unified design isn't just important for the sake of a great user experience
+across multiple devices. Right now, 13% of US adults are smartphone-only users,
+and that percentage rises for people without a college degree, African Americans,
```

Comment:
Hey @AnneTee - a couple of thoughts on this statistic: I think if you read it quickly as `and that percentage rises for people without a college degree, African Americans, people who make less than $20,000 a year` it could come across as negatively describing black people. Also, I'm not sure if `African American` is preferable to black, unless that is what the stats specifically say. It could also be worth citing the stats directly in this case. I'm honestly not sure, but it's worth taking a closer look.